integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/mortgage/MortgageSpark.scala

```diff
@@ -430,8 +430,17 @@ object Main {
       .appName("MortgageJob")
       .getOrCreate()
 
-    0.until(10).foreach { _ =>
-      Run.parquet(session, perfPath, acqPath).write.mode("overwrite").parquet(output)
+    // extend args to support csv/orc/parquet dataset
+    if (args.length > 5) {
+        throw new IllegalArgumentException("Supports up to 5 arguments.")
     }
+    val format = args.lift(4).getOrElse("parquet")
+    val runFun = format match {
+      case "csv" => Run.csv(session, perfPath, acqPath)
+      case "orc" => Run.orc(session, perfPath, acqPath)
+      case _ => Run.parquet(session, perfPath, acqPath)
```

Comment:
This shouldn't accept any string as Parquet, otherwise if the user specifies "json" they're going to be surprised.
```suggestion
      case "parquet" => Run.parquet(session, perfPath, acqPath)
      case _ =>
          System.err.println(s"Invalid input format $format, expected one of csv, orc, parquet")
          System.exit(1)
```

Reply:
> This shouldn't accept any string as Parquet, otherwise if the user specifies "json" they're going to be surprised.

Updated